### DISCUSSION TITLE
MonitorControl v3.1.0

### DISCUSSION THREAD

**Discussion by JoniVR** [MonitorControl/MonitorControl#619] [Release Discussion]
### Enhancements
- Add a setting to control the function of F14 / F15 keys, this defines if they should be used as alternate brightness keys or not (#602)
- Updated/Added French, Italian, Japanese, German, Chinese, Dutch translations

### Notes
- This is a smaller, stable release, `v4.0.0` is already very much in development (see #596), with another huge amount of features. We're going to be starting another round of beta releases in the future to make sure `v4.0.0` is stable and easy-to-use before fully releasing it. 
- Thanks to everyone who helped testing, contributed and donated! It really helps a lot!

<hr /><em>This discussion was created from the release <a href='https://github.com/MonitorControl/MonitorControl/releases/tag/v3.1.0'>MonitorControl v3.1.0</a>.</em>

**Comment by Valrani** [comment]
Hello, great soft :)
Is it possible to remap the keyboard shortcut, as I use a keyboard without F14/F15 keys ? I'd like to put F9/F10 for example.
Sorry but I'm not familiar with GitHub I don't know where to post.

**Reply by waydabber** [reply]
Hi Valrani, custom shortcuts will be added to 4.0.0 soon. Until that time you need to use a third party app like Karabiner to remap the keys.
